modules/local/genomescope/main.nf

```diff
@@ -0,0 +1,31 @@
+process GENOMESCOPE {
```

Comment:
Why not the nf-core module?

Reply:
The nf-core module does not emit what I need, the estimated length.